### PR TITLE
Feat/horse details

### PR DESCRIPTION
**Pull request by augustreid**
- [x] - Feature PR [🧩]
- [ ] - Refactor PR [⚙️]
- [ ] - BugFix PR [🐞]
- [ ] - User Interface PR [🖍]
- [ ] - Error Handling PR [⚠️]
- [ ] - Testing PR [🧑‍💻]

**What (if any) features are you implementing?**

 - The AllHorses component to display horse cards
 - The HorseCard component
 - The HorseProfile component to display all horse details, including a link back to AllHorses

**What (if anything) did you refactor?**

 - (NA)

**Were there any issues that arose?**

 - I couldn't get the images working, but didn't spend much time on it since we will need different images anyway

**Is there anything needs to be prioritized/errors to be addressed?**

 - There will definitely be some refactoring once we start using the API

**Goals? Next steps?**

 - Keep on truckin'!

### CODE REVIEW

File: src/HorseProfile/HorseProfile.js [hunk 1]
```diff
@@ -1,6 +1,93 @@
+import React from "react";
+import { useParams, Link } from "react-router-dom";
+import mockData from "../mockData";
+
 function HorseProfile() {
+  const horseId = Number(useParams().id);
+  let horse;
```
Comment (jkiernan12) [inline src/HorseProfile/HorseProfile.js:7]: Using let and const for horse, owner, vet and farrier definitely works for now, but we may need to use state later on if we add an 'edit' button to this page.

File: src/mockData.js [hunk 2]
```diff
@@ -123,4 +126,6 @@
     }
     ]
   }
-}
\ No newline at end of file
+}
+
+export default mockData;
```
Comment (jkiernan12) [inline src/mockData.js:131]: ^^ Took me forever to realize why I couldn't use mockData in my branch until I saw this lol

File: src/HorseProfile/HorseProfile.js [hunk 3]
```diff
@@ -1,6 +1,93 @@
...
+    return horse.vet_id === oneVet.id;
+  })
+
+  const farrier = mockData.data.farriers.find((oneFarrier) => {
+    return horse.farrier_id === oneFarrier.id;
+  })
+
   return ( 
-    <p>Horse profile here</p>
+    <div>
```
Comment (jkiernan12) [inline src/HorseProfile/HorseProfile.js:28]: This page looks great!

**Review by jkiernan12 — APPROVED** [review]
Great work! All the info is there -- nice job!

**Review by ewang0 — APPROVED** [review]
Looks good! I have some images for mock data if we need them.

**Pull request merged**